lib/money/rates_store/memory.rb

```diff
@@ -88,20 +89,18 @@ def transaction(&block)
       #     puts [iso_from, iso_to, rate].join
       #   end
       def each_rate(&block)
```

Comment:
No need to accept `&block` any more if you're just going to yield instead of passing the block to the Enumerator.